data/human/transport missions.txt

```diff
@@ -3582,3 +3582,732 @@ mission "Hauler VI cargo prototype 3"
+
+
+
+
+	name "Take family home to <planet>"
+	description "You meet a family worried about the potential destruction of war in the Free Worlds and want to cut their vacation short and go home to <destination>."
+	source
+		government "Free Worlds"
+		attributes "tourism"
+	destination "Mainsail"
+	passengers 3
+	cargo "personal belongings" 1
+	to offer
+		has "FW Katya 7C: done"
+	to fail
+		has "free worlds plot completed"
+	on offer
+		conversation
+			`After walking around the spaceport for awhile you head back to your ship to find and middle aged couple with their two grade school daughters waiting for you at your hanger. The man is busy trying to adjudicate a dispute between the girls but the lady notices you immediately and begins to approach you. "Thank goodness you're here," she says. "We were hoping you could get us back home to the paradise planets. This war caught us right in the middle of our vacation and we can't risk being anywhere near here when the Navy attacks, especially with our little 'angels'," she gestures toward her daughters, now in a full blown fight. "If you can find the time to take us we will pay you very well."`
+			choice
+				`	"Sure, I'll take you."`
+				`	"OK. But it's pretty far. It better be worth it."`
+				`	"No. I don't think so. Sorry."`
+					decline
+			`	"Then it's a deal," she shakes your hand firmly. The whole family gets onto the <ship> without another word, leaving you to load their many bags and cases.`
+				accept
+
+	on complete
+		payment 66999
+		conversation
+			`You're not sure if you have ever met a more pretentious and entitled family. The girls fought non-stop and both they all complained constantly of the lack of luxury accommodation on your ship. You are very happy to finally be free of them. Although you feel you should have been paid more for how terrible they were.`
+
+	on visit
+		dialog phrase "generic cargo and passenger on visit"
+
+
+
+mission "Journalism 1"
+	name "Transport exposition to <planet>"
+	description `Transport expo equipment to <destination>.`
+	minor
+	source
+		attributes "paradise"
+		not attributes "mining"
+		not attributes "textiles"
+		not attributes "research"
+		not attributes "factory"
+	destination "Moonshake"
+	cargo "exposition installations" 5
+	to offer
+		random < 80
+		has "Expedition to Hope 2: done"
+		has "FW Katya 4: done"
+
+	on offer
+		conversation
+			`As you explore the luxurious spaceport you notice a large group of people around one of the larger hangers.`
+			choice
+				`	(Check it out.)`
+				`	(Not right now.)`
+					defer
+			`	You approach some of the people at the end of the line and ask what's going on. "It's the last day of Adelita's exposition on the evacuation of Hope," says an excited young woman. "It was their first work that really got them famous. The last week of their expositions are always free. This is a great chance to see it if you haven't already."`
```

Comment:
A. Lot. Of. Periods. Dialogue needs revision.

Reply:
Im not sure if three periods in a paragraph of dialog is a lot, but im a fan of Vonnegut. I will get change on to a comma.